src/components/QueryCondition.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+	<div class="querybuilder__rule" :fieldErrorsrrr="fieldErrorsrrr">
+		<PropertyLookup
+			v-model="selectedProperty"
+			:error="fieldErrors.property"
+		/>
+		<ValueTypeDropDown
+			v-model="selectedPropertyValueRelation"
+			:disabled="limitedSupport"
+		/>
+		<TextInput
+			class="querybuilder__rule__value"
+			:label="$i18n('query-builder-input-value-label')"
+			ref="value"
+			v-model="textInputValue"
+			:error="fieldErrors.value ?
+				{message: $i18n(fieldErrors.value.message), type: fieldErrors.value.type}: null"
+			:placeholder="$i18n('query-builder-input-value-placeholder')"
+			:disabled="selectedPropertyValueRelation === propertyValueRelation.Regardless"
+		/>
+	</div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { mapState } from 'vuex';
+import { TextInput } from '@wmde/wikit-vue-components';
+
+import PropertyLookup from '@/components/PropertyLookup.vue';
+import ValueTypeDropDown from '@/components/ValueTypeDropDown.vue';
+import SearchResult from '@/data-access/SearchResult';
+import PropertyValueRelation from '@/data-model/PropertyValueRelation';
+import Error from '@/data-model/Error';
+import allowedDatatypes from '@/allowedDataTypes';
+import Validator from '@/form/Validator';
+
+export default Vue.extend( {
+	name: 'QueryCondition',
+	data() {
+		return {
+			fieldErrors: {
+				property: null as null | Error,
+				value: null as null | Error,
+			},
+			limitedSupport: false,
+			propertyValueRelation: PropertyValueRelation,
```

Comment:
As far as I can tell, this is unused can just be dropped.

Reply:
Done

Reply:
It is used in the template: propertyValueRelation.Regardless, when I removed it, lots of tests failed with:
 ● QueryCondition.vue › updates the store value when the user fills in the value textfield

    TypeError: Cannot read property 'Regardless' of undefined

Reply:
That should be `PropertyValueRelation.Regardless` (capital `P`, the thing that is imported)

Reply:
That doesn't work, I think inside template, you can't do it. I can extract it to a method and use that instead.

Reply:
> I can extract it to a method and use that instead.

Yes, a dedicated Method determining whether the Dropdown is disabled seems less confusing than a state that is actually just used as an enum.

Reply:
Done

Reply:
:+1: